src/components/ConversationSettings/BotsSettings.vue

```diff
@@ -76,21 +76,19 @@ export default {
 	},
 
 	setup() {
-		const botsStore = useBotsStore()
-
 		return {
-			botsStore,
+			botsStore: useBotsStore(),
 		}
 	},
```

Comment:
![image](https://github.com/nextcloud/spreed/assets/84044328/4ffaffe1-8983-49c1-907e-6cbf66f0ad59)

I think ts in SFC needs more refactoring, methods used in template are not inferred only when they are added to setup hook.